runtime/Cpp/runtime/src/TokenStreamRewriter.cpp

```diff
@@ -358,7 +357,7 @@ std::unordered_map<size_t, TokenStreamRewriter::RewriteOperation*> TokenStreamRe
         rop->lastIndex = std::max(prevRop->lastIndex, rop->lastIndex);
         std::cout << "new rop " << rop << std::endl;
       }
-      else if (!disjoint && !same) {
+      else if ( !disjoint ) {
```

Comment:
Please honor the existing formatting style. No spaces around expressions in parentheses.